src/dbs.js

```diff
@@ -1,33 +1,41 @@
+const request = require('request-promise-native'),
+      {URL} = require('url');
+
 const PouchDB = require('pouchdb-core');
 PouchDB.plugin(require('pouchdb-adapter-http'));
 PouchDB.plugin(require('pouchdb-mapreduce'));
 
 const STAGING_URL = 'https://staging.dev.medicmobile.org/_couch/builds';
 
-if (process.env.TESTING) {
-  const unmocked = () => Promise.reject('this needs to be mocked');
 
+if (process.env.TESTING) {
   module.exports = {
-    app: {
-      allDocs: unmocked,
-      bulkDocs: unmocked,
-      get: unmocked,
-      put: unmocked,
-      query: unmocked,
-      remove: unmocked
-    },
-    builds: {
-      get: unmocked,
-      put: unmocked,
-      query: unmocked
-    }
+    app: {},
+    builds: {},
   };
 } else {
-  const COUCH_URL = process.env.COUCH_URL;
-  if(!COUCH_URL) throw new Error('COUCH_URL env var not set.');
+  const COUCH_URL = new URL(process.env.COUCH_URL);
+  COUCH_URL.pathname = '/';
+
+  const activeTasks = () => {
+    return request({
+      url: COUCH_URL + '/_active_tasks',
+      json: true
+    }).then(tasks => {
+      // TODO: consider how to filter these just to the active database.
+      // On CouchDB 2.x you only get the shard name, which looks like:
+      // shards/80000000-ffffffff/medic.1525076838
+      // On CouchDB 1.x (I think) you just get the exact DB name
+      return tasks;
+    });
+  };
+
+  const DEPLOY_URL = process.env.COUCH_URL;
+  if(!DEPLOY_URL) throw new Error('COUCH_URL env var not set.');
 
   module.exports = {
-    app: new PouchDB(COUCH_URL),
-    builds: new PouchDB(STAGING_URL)
+    app: new PouchDB(DEPLOY_URL/*, {ajax: {timeout: 60000}}*/),
+    builds: new PouchDB(STAGING_URL),
+    active_tasks: activeTasks
```

Comment:
This should be `activeTasks: activeTasks`. Just like it's `bulkDocs` not `bulk_docs`.